tests/test_optics.py

```diff
@@ -66,159 +66,159 @@
+#     psf *= dx_test**2 # put the PSF into flux units rather than SB for comparison
+#     mtf_test = np.abs(np.fft.fft2(psf))
+#     # Compare
+#     mtf = galsim.optics.mtf(array_shape=testshape, scale=dx_test, lam_over_diam=lod_test)
+#     np.testing.assert_array_almost_equal(mtf, mtf_test, decimal=decimal_dft)
+#     t2 = time.time()
+#     print 'time for %s = %.2f'%(funcname(),t2-t1)
+
+# def test_wavefront_image_view():
+#     """Test that the ImageF.array view of the wavefront is consistent with the wavefront array.
+#     """
+#     import time
+#     t1 = time.time()
+#     array, _ = galsim.optics.wavefront(array_shape=testshape)
+#     (real, imag), _ = galsim.optics.wavefront_image(array_shape=testshape)
+#     np.testing.assert_array_almost_equal(array.real.astype(np.float32), real.array, decimal)
+#     np.testing.assert_array_almost_equal(array.imag.astype(np.float32), imag.array, decimal)
+#     t2 = time.time()
+#     print 'time for %s = %.2f'%(funcname(),t2-t1)
+
+# def test_psf_image_view():
+#     """Test that the ImageF.array view of the PSF is consistent with the PSF array.
+#     """
+#     import time
+#     t1 = time.time()
+#     array, _ = galsim.optics.psf(array_shape=testshape)
+#     image = galsim.optics.psf_image(array_shape=testshape)
+#     np.testing.assert_array_almost_equal(array.astype(np.float32), image.array, decimal)
+#     t2 = time.time()
+#     print 'time for %s = %.2f'%(funcname(),t2-t1)
+
+# def test_otf_image_view():
+#     """Test that the ImageF.array view of the OTF is consistent with the OTF array.
+#     """
+#     import time
+#     t1 = time.time()
+#     array = galsim.optics.otf(array_shape=testshape)
+#     (real, imag) = galsim.optics.otf_image(array_shape=testshape)
+#     np.testing.assert_array_almost_equal(array.real.astype(np.float32), real.array, decimal)
+#     np.testing.assert_array_almost_equal(array.imag.astype(np.float32), imag.array, decimal)
+#     t2 = time.time()
+#     print 'time for %s = %.2f'%(funcname(),t2-t1)
+
+# def test_mtf_image_view():
+#     """Test that the ImageF.array view of the MTF is consistent with the MTF array.
+#     """
+#     import time
+#     t1 = time.time()
+#     array = galsim.optics.mtf(array_shape=testshape)
+#     image = galsim.optics.mtf_image(array_shape=testshape)
+#     np.testing.assert_array_almost_equal(array.astype(np.float32), image.array)
+#     t2 = time.time()
+#     print 'time for %s = %.2f'%(funcname(),t2-t1)
+
+# def test_ptf_image_view():
+#     """Test that the ImageF.array view of the OTF is consistent with the OTF array.
+#     """
+#     import time
+#     t1 = time.time()
+#     array = galsim.optics.ptf(array_shape=testshape)
+#     image = galsim.optics.ptf_image(array_shape=testshape)
+#     np.testing.assert_array_almost_equal(array.astype(np.float32), image.array)
+#     t2 = time.time()
+#     print 'time for %s = %.2f'%(funcname(),t2-t1)
```

Comment:
I think I'm ok with actually deleting the above tests rather than merely commenting them out.  

Or, even better, you could move them to test_deprecated and check that using the old OpticalPSF class raises deprecation warnings.  For things like mtf, otf, etc. that don't have new analogs, that warning should tell people that if they need those functions to add an issue about it, since we aren't planning on supporting them anymore.  (I doubt anyone will care, but that would be the polite thing to do I think.)